python/tests/parsers/test_parser_class.py

```diff
@@ -52,32 +57,36 @@ def new_parser(filing_detail_text_13f, primary_doc_xml_text, infotable_xml_text)
         return Parser('')
 
 
-PARSER = new_parser(filing_detail_text_13f(), primary_doc_xml_text(), infotable_xml_text())
+@pytest.fixture
```

Comment:
Set the scope so that this isn't created for each test.

https://docs.pytest.org/en/6.2.x/fixture.html#scope-sharing-fixtures-across-classes-modules-packages-or-session